src/merge.js

```diff
@@ -19,6 +19,9 @@ var _extend = require('./internal/_extend');
  *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': 40 });
  *      //=> { 'name': 'fred', 'age': 40 }
  *
+ *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': undefined });
+ *      //=> { 'name': 'fred', 'age': undefined }
```

Comment:
> The purpose of this commit is to clarify the behavior of `R.merge` with respect to any items of higher precedence that contain fields set to `undefined`.

I'm curious as to why one might expect `undefined` to be treated differently from other values. Perhaps some other libraries have set a bad precedent. ;)